RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletHeterogeneousEDProducer.cc

```diff
@@ -98,45 +98,42 @@ void CAHitNtupletHeterogeneousEDProducer::beginStreamGPUCuda(
 void CAHitNtupletHeterogeneousEDProducer::acquireGPUCuda(
     const edm::HeterogeneousEvent &iEvent, const edm::EventSetup &iSetup,
     cuda::stream_t<> &cudaStream) {
+
+  seedingHitSets_ = std::make_unique<RegionsSeedingHitSets>();
+
+
+  // FIXME: move directly to region or similar...
   edm::Handle<IntermediateHitDoublets> hdoublets;
```

Comment:
I think I don't have anything else to comment (in addition to the list in the description), except "Yes!".

As discussed with @felicepantaleo and @fwyzard earlier, for the GPU workflow we definitively need to drop the "CPU doublets" (and so this `get`), and also move the "seeding layer" configuration as part of the GPU CA producer to drop the `SeedingLayersEDProducer` from the sequence.